script.js

```diff
@@ -298,4 +317,26 @@ formTwo.addEventListener('submit', (event) => {
   }
 });
 
-// end validate email
\ No newline at end of file
+// end validate email
+
+// store information in storage
+
+mobileNameInput.addEventListener('keyup', () => {
+  localStorage.setItem('mobileNameInput', mobileNameInput.value);
+});
+mobileEmailInput.addEventListener('keyup', () => {
+  localStorage.setItem('mobileEmailInput', mobileEmailInput.value);
+});
+mobileMessageInput.addEventListener('keyup', () => {
+  localStorage.setItem('mobileMessageInput', mobileMessageInput.value);
+});
+
+desktopNameInput.addEventListener('keyup', () => {
+  localStorage.setItem('mobileNameInput', desktopNameInput.value);
+});
+desktopEmailInput.addEventListener('keyup', () => {
+  localStorage.setItem('mobileEmailInput', desktopEmailInput.value);
+});
+desktopMessageInput.addEventListener('keyup', () => {
+  localStorage.setItem('mobileMessageInput', desktopMessageInput.value);
+});
```

Comment:
- According to the project requirements, `The form data for all inputs should be stored in a single object, saved to a local storage registry`. Currently, I noticed you are saving each input value differently. Please fix this 🙏  
<img width="958" alt="rjsyjsrj" src="https://user-images.githubusercontent.com/73607512/218282691-f8e6f8fb-b205-47e0-af2d-1f0e11ee0e66.PNG">